code/modules/vehicles/armored/ammo_magazine.dm

```diff
@@ -52,6 +52,20 @@
 	max_rounds = 200
 	magazine_flags = MAGAZINE_REFILLABLE|MAGAZINE_NOT_FABRICABLE
 
+/obj/item/ammo_magazine/tank/autocannon
+	name = "Bushwhacker Autocannon APDS Box (30mm)"
+	desc = "A 200 round box for an autocannon. Loaded with Armor Piercing rounds."
+	caliber = CALIBER_30X17MM
+	icon_state = "tank_autocannon_ap"
+	max_rounds = 250
+	default_ammo = /datum/ammo/bullet/tank_autocannon
```

Comment:
gonna leave this to kuro but uh, xenos are sure gonna have an interesting time against 250 grenade magazines that can be rapid fired

Reply:
Gonna rework it into just firing fragmentation 360. I put down high explosive as explosions since that was the first thing that came to mind